### ISSUE TITLE
Tkinter on Macos: 

### ISSUE THREAD

**Issue by brenthuisman** [pypy/pypy#5082]
When I try to `import tkinter` using the [Macos ARM build](https://downloads.python.org/pypy/pypy3.10-v7.3.17-macos_arm64.tar.bz2), I am greeted with the following error:

```
Python 3.10.14 (39dc8d3c85a7, Aug 27 2024, 20:40:24)
[PyPy 7.3.17 with GCC Apple LLVM 15.0.0 (clang-1500.3.9.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>>> import tkinter
Traceback (most recent call last):
  File "<python-input-0>", line 1, in <module>
    import tkinter
  File "/Users/brent/Downloads/pypy3.10-v7.3.17-macos_arm64/lib/pypy3.10/tkinter/__init__.py", line 42, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
    ^^^^^^^^^^^^^^^
  File "/Users/brent/Downloads/pypy3.10-v7.3.17-macos_arm64/lib/pypy3.10/_tkinter/__init__.py", line 14, in <module>
    from .tklib_cffi import ffi as tkffi, lib as tklib
ImportError: dlopen(/Users/brent/Downloads/pypy3.10-v7.3.17-macos_arm64/lib/pypy3.10/_tkinter/tklib_cffi.pypy310-pp73-darwin.so, 0x0006): Library not loaded: /opt/homebrew/opt/tcl-tk/lib/libtcl8.6.dylib
  Referenced from: <4F3F66C4-244C-3384-B6AE-32D999519DF3> /Users/brent/Downloads/pypy3.10-v7.3.17-macos_arm64/lib/pypy3.10/_tkinter/tklib_cffi.pypy310-pp73-darwin.so
  Reason: tried: '/opt/homebrew/opt/tcl-tk/lib/libtcl8.6.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/tcl-tk/lib/libtcl8.6.dylib' (no such file), '/opt/homebrew/opt/tcl-tk/lib/libtcl8.6.dylib' (no such file)

```

In the release notes for 7.3.11 I read however that Tkinter is now supplied for Macos builds[1]. I can find a single non-`.py` file when grepping for 'tk': `lib/pypy3.10/_tkinter/tklib_cffi.pypy310-pp73-darwin.so`. Is there anything extra I should be doing?

[1] https://doc.pypy.org/en/latest/release-v7.3.11.html?highlight=tkinter#bugfixes

**Comment by mattip** [comment]
I think you need to `brew install x11` to be able to use tkinter. How does it work for you with the CPython installs?

**Comment by mattip** [comment]
Sorry, `brew install libx11` is the correct recipe.

**Comment by brenthuisman** [comment]
I cannot use `brew` due to company policy (don't ask). I am surprised that the pre-built tarball still requires brew to install a dependency though. If it is at all possibly to supply the dependency, it would be great.

**Comment by mattip** [comment]
I agree it is strange we don't bundle it. I see the package script [should pick up](https://github.com/pypy/pypy/blob/main/pypy/tool/release/make_portable.py#L23) the location of `libtk*` and `libtcl*`:
```
% otool -L build-worker-arm64/pypy-c-jit-macos-arm64/build/lib_pypy/_tkinter/tklib_cffi.pypy310-pp73-darwin.so 
build-worker-arm64/pypy-c-jit-macos-arm64/build/lib_pypy/_tkinter/tklib_cffi.pypy310-pp73-darwin.so:
        ./_tkinter/tklib_cffi.pypy310-pp73-darwin.so (compatibility version 0.0.0, current version 0.0.0)
        /opt/homebrew/opt/tcl-tk/lib/libtcl8.6.dylib (compatibility version 8.6.0, current version 8.6.14)
        /opt/homebrew/opt/tcl-tk/lib/libtk8.6.dylib (compatibility version 8.6.0, current version 8.6.14)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1345.120.2)
```

I will mark this for the next release to investigate what is going on an trying to solve it.

**Comment by brenthuisman** [comment]
Thanks Matti!

**Comment by mattip** [comment]
Fixed in 99cdfd711d. You can try the next nightly build.

**Comment by mattip** [comment]
Closing, please reopen or open a new issue if there is still something missing
